Hover/Model/HoverConfiguration.swift

```diff
@@ -54,7 +56,8 @@ public struct HoverConfiguration {
                 font: UIFont? = nil,
                 dimColor: UIColor = UIColor.black.withAlphaComponent(0.75),
                 initialPosition: HoverPosition = .bottomRight,
-                allowedPositions: Set<HoverPosition> = .all) {
+                allowedPositions: Set<HoverPosition> = .all,
+                imageExpandAnimation: ImageExpandAnimation = .none) {
```

Comment:
🤔  Can we move `imageExpandAnimation` to be right below` image`? I think it might look and read a bit better.